dspace/modules/xmlui-mirage2/pom.xml

```diff
@@ -89,6 +89,7 @@
                                         <include>**/*.ttf</include>
                                         <include>**/*.woff</include>
                                         <include>**/*.eot</include>
+                                        <include>**/*.xsl</include>
```

Comment:
This, and the few lines above are very odd.  There's no such thing as an `<include>` within an `<excludes>` tag.  These should all say `<exclude>`, I believe.
https://maven.apache.org/plugins/maven-resources-plugin/examples/include-exclude.html